Common/UInt40.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace EnlightenMAUI.Common
+{
+    /// <summary>
+    /// 40-bit unsigned value, used for many of the laser functions.
+    /// </summary>
+    [ComVisible(true)]
+    [Guid("5BC91277-C373-44F3-8C84-77964365F627")]
+    [InterfaceType(ComInterfaceType.InterfaceIsDual)]
+    public interface IUInt40
```

Comment:
For the record, SiG FPGAs use UInt32 for all modulation commands; there are no 40-bit parameters.

That said, the "extra byte" was necessarily passed as the first byte of the historically-mandatory 8-byte payload block, and is uniformly zero in all anticipated use-cases, which is the default payload value _anyway,_ so there is probably no consequence of continuing to "pretend" these are 40-bit fields on SiG. Other than confusing readers who treat this as current reference code.

Reply:
I thought this class was a dependency for another one, but it ended up being for a feature we aren't currently supporting. I'll delete for now and bring back over if we need it